typescript/packages/common-runner/src/builtins/stream-data.ts

```diff
@@ -0,0 +1,101 @@
+import { type Node } from "@commontools/common-builder";
+import { cell, CellImpl, ReactivityLog } from "../cell.js";
+import { sendValueToBinding, findAllAliasedCells } from "../utils.js";
+import { schedule, Action } from "../scheduler.js";
+import { mapBindingsToCell } from "../utils.js";
+
+/**
+ * Stream data from a URL.
+ *
+ * Returns the streamed result as `result`. `pending` is true while a request is pending.
+ *
+ * @param url - A cell containing the URL to stream data from.
+ * @returns { pending: boolean, result: any, error: any } - As individual cells, representing `pending` state, streamed `result`, and any `error`.
+ */
```

Comment:
Explain why this exists better, document the generator and explain how reactivity could work.